wallet/cardano-sl-wallet.cabal

```diff
@@ -1,5 +1,5 @@
 name:                cardano-sl-wallet
-version:             1.1.0
+version:             1.1.1
```

Comment:
@rvl Should be version-bump `cardano-sl-wallet-new` as well?

Reply:
I missed it in the release branch because my sed command was `s/1\.1\.0/1.1.1/`.
Maybe let's just fix it when bumping to 1.2.0 (this can be done on develop branch -- no need to wait for a release branch to be cut).